### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by anapaulagomes**
This PR adds more information about the project including a small demo.
Solves https://github.com/anapaulagomes/pytest-picked/issues/6.

### CODE REVIEW

**Comment by anapaulagomes** [comment]
@MisterRios, can you take a look on it, please? :)

**Comment by MisterRios** [comment]
@anapaulagomes Sure thing. Will take a look tonight.

File: README.rst [hunk 1]
```diff
@@ -10,48 +10,95 @@ pytest-picked
...
 ------------
 Contributions are very welcome. Tests can be run with `tox`_, please ensure
 the coverage at least stays the same before you submit a pull request.
 
+
 License
 -------
 
-Distributed under the terms of the `MIT`_ license, "picked" is free and open source software
+Distributed under the terms of the `MIT`_ license, "pytest-picked" is free and open source software
```
Comment (MisterRios) [inline README.rst]: :+1:

File: README.rst [hunk 2]
```diff
@@ -10,48 +10,95 @@ pytest-picked
...
-`pytest --picked`
+::
 
+  $ pytest --picked
 
-TODO
------
 
-* Add option to run the changed files from the current branch
-* Guess the test file from given changed file
```
Comment (MisterRios) [inline README.rst]: Ah, great to remove these. I was confused when I first read these. Glad you moved them into issues.

File: README.rst [hunk 3]
```diff
@@ -10,48 +10,95 @@ pytest-picked
...
+---
 
+.. image:: demo.gif
+    :target: https://asciinema.org/a/cWV3sx6poG4aaPZfyVI8nOJ7J
+    :height: 400px
+    :alt: Demo
 
-Features
---------
+Let's say you have the following output from your ``git status``:
```
Comment (MisterRios) [inline README.rst]: Maybe remove the `your`:
`Let's say you have the following output from ``git status``:`

File: README.rst (hunk 1 above)
Comment (MisterRios) [reply README.rst]: Thanks for making all of these changes. They describe the package a lot better!

File: README.rst [hunk 4]
```diff
@@ -10,48 +10,95 @@ pytest-picked
...
+  tests/test_board.py .                                                      [ 50%]
+  tests/api/test_new.py .                                                    [100%]
+
+  =========================== 2 passed in 0.07 seconds ===========================
+
+The files and folders with tests, that were modified but weren't committed yet will be run.
+No more copy and paste!
 
 
 Usage
```
[the remark this answers is not in the document]
Comment (anapaulagomes) [reply README.rst]: I wondered the same but I'd rather keep the _Usage_ section with the commands available.

**Comment by anapaulagomes** [comment]
Thanks for reviewing it @MisterRios ! :) I'll change the demo gif later.

File: README.rst [hunk 5]
```diff
@@ -10,48 +10,95 @@ pytest-picked
...
+    :target: https://pypi.org/project/pytest-picked/
+    :alt: See Package Status on PyPI
 
-----
+Run the tests related to the modified files (according to Git)
 
-This `pytest`_ plugin was generated with `Cookiecutter`_ along with `@hackebrot`_'s `cookiecutter-pytest-plugin`_ template.
+---
 
+.. image:: demo.gif
```
[the remark this answers is not in the document]
Comment (anapaulagomes) [reply README.rst]: @MisterRios I've updated the demo. Can you take a look, please?

[the remark this answers is not in the document]
Comment (MisterRios) [reply README.rst]: Looks great!

**Pull request merged**